config/routes.rb

```diff
@@ -39,7 +40,6 @@
 
   get "search/", to: "search#search"
 
-  resources :tickets, only: %i[new create]
   get "tickets/confirmation", to: "tickets#confirmation"
```

Comment:
I'd also remove this route - and add a redirect route for `https://www.data.gov.uk/tickets/new` -> `https://www.data.gov.uk/support` 🙏 thank you!

Reply:
Does this look all right?

Reply:
(Line 43 still needs deleting)